README.rst

```diff
@@ -1,26 +1,31 @@
 Responses
 =========
 
-.. image:: https://travis-ci.org/getsentry/responses.svg?branch=master
-	:target: https://travis-ci.org/getsentry/responses
+..  image:: https://travis-ci.org/getsentry/responses.svg?branch=master
```

Comment:
quick note here, that generally people don't actually try to line up these indents as its quite painful. i'm going to merge the change though